src/Api/Endpoints/Magic.cs

```diff
@@ -38,22 +39,31 @@ public static void MapMagicEndpoints(this IEndpointRouteBuilder app)
             .RequireSecretKey()
             .WithTags(OpenApiTags.MagicLinks);
 
-        group.MapPost("/send", async (
-                SendMagicLinkRequest request,
-                IFeatureContextProvider provider,
-                MagicLinkService magicLinkService
-            ) =>
-            {
-                if (!(await provider.UseContext()).IsMagicLinksEnabled)
-                    throw new ApiException("You must enable Magic Links in settings in order to use this feature.", 403);
-
-                await magicLinkService.SendMagicLinkAsync(request.ToDto());
-
-                return NoContent();
-            })
+        group.MapPost("/send", SendMagicLinkAsync)
             .WithParameterValidation()
-            .RequireSecretKey()
-            .RequireCors("default")
             .RequireRateLimiting(RateLimiterPolicy);
     }
+
+    /// <summary>
+    /// Sends email containing a magic link template allowing users to login.
```

Comment:
Technically the email contains a magic link, not a template (the template tokens are resolved before the email is sent).

```suggestion
    /// Sends an e-mail containing a magic link allowing the specified user to login.
```